tests/functional/Glpi/Inventory/Assets/Drive.php

```diff
@@ -457,19 +457,19 @@ public function testInventoryUpdateHardDrive()
 
        //we still have 3 harddrives
         $harddrives = $device_hdd->find();
-        $this->integer(count($harddrives))->isIdenticalTo(3);
+        $this->integer(count($harddrives))->isIdenticalTo(5);
```

Comment:
Why 5 hardrive now?

Reply:
All changes in `testInventoryUpdateHardDrive` seems wrong.